health/health.go

```diff
@@ -0,0 +1,78 @@
+package health
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+type healthStatus struct {
+	PostgresDb string `json:"postgres_db"`
+	MongoDb    string `json:"mongo_db"`
+}
+
+type HandlerHealth struct {
+	PostgresDb *pgxpool.Pool
+	MongoDB    bool
```

Comment:
This is a bad approach, once you have a new database or want to delete one, you need to go here and cleanup the code.  The health check should know nothing about what exactly checks it has. Check should be an abstraction.